### PR TITLE
Fixed a bug where zips were not removed after unzip

### PR DESCRIPTION
**Pull request by falfiya**
## Description
Now the app removes the zip files that are downloaded.

## Motivation and Context

Saves space.

## How Has This Been Tested?

I've run it with Visual Studio

## Screenshots (if appropriate):
image

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

I think that the Visual Studio auto format changed a good bit of the code. I named things mostly camelCase

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by skid9000** [comment]
The PR should cover ONLY what you said, if you want to refactor the code, please open a separate PR.

File: Crunchyroll-Downloader/App.config [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.1"/>
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/>
```
Comment (skid9000) [inline Crunchyroll-Downloader/App.config:4]: Please, don't downgrade the .NET Framework version.

File: Crunchyroll-Downloader/Crunchyroll-Downloader.csproj [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Crunchyroll_Downloader</RootNamespace>
     <AssemblyName>Crunchyroll-Downloader</AssemblyName>
-    <TargetFrameworkVersion>v4.7.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```
Comment (skid9000) [inline Crunchyroll-Downloader/Crunchyroll-Downloader.csproj:12]: Please, don't downgrade the .NET Framework version.

**Review by skid9000 — COMMENTED** [review]
This is just too much of a mess, please open a new PR, one for the new feature, one for the refactor.
And BTW, no indent with space.

File: Crunchyroll-Downloader/App.config (hunk 1 above)
Comment (falfiya) [reply Crunchyroll-Downloader/App.config:4]: Oops. Sorry. I didn't see that.

**Pull request closed without merging**